consensus/subworkflows.nf

```diff
@@ -42,21 +46,17 @@ include {
 workflow CygnusConsensus {
     take:
         reads_fastq
-
     main:
         Cygnus(reads)
-        RotateBySequence(Cygnus.out)
```

Comment:
Most other Cygnus workflows include process `RotateBySequence`. Is its removal here intended?

Reply:
Yes, aim is to facilitate just running the consensus algo's.